src/server/graphql/schemas/Cycle.js

```diff
@@ -19,6 +19,7 @@ export default new GraphQLObjectType({
       updatedAt: {type: new GraphQLNonNull(GraphQLDateTime), description: 'The time when the cycle was last updated'},
       state: {type: CycleState, description: 'What state the cycle is currently in'},
       chapter: {type: Chapter, description: 'The chapter', resolve: resolveChapter},
+      startOfWeek: {type: StartOfWeek, description: 'The first day of the cycle', resolve: resolveStartOfWeek},
```

Comment:
As noted, let's keep this simple in the API and just make the `weekStartedAt` field type `GraphQLDateTime`. It can still use a custom reolver function that returns just a regular JS date (using `moment`'s `.toDate()` method).